src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -99,6 +87,13 @@ public void execute_filteredList_success() {
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_noFieldSpecifiedUnfilteredList_failure() {
```

Comment:
Grouping the positive and negative test cases allows for better readability. Good job!